src/Controller/CompanyController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class CompanyController extends AbstractController
+{
+    #[Route('/company', name: 'app_company')]
```

Comment:
It is not recommended to use routes.yank and annotations for declaring the routes. In this case the show method wull match 2 routes:
   - routes.yaml: /company/show
   - annotation: /company
This adds unnecessary complexity. If you wish to have a route company/ you can add a it in routes.yaml
To see allthe available routes run in project folder:
![image](https://user-images.githubusercontent.com/19302124/202656143-5d5e53f0-1597-4598-8694-ea7741c52d9a.png)

You can see that '/company' and '/company/show' are 2 different routes, but in the in the code they point to the same controller function.

We will use the yaml comfiguration so the annotation can be removed. Alse remove line 7.